cmd/helm-operator/main.go

```diff
@@ -28,10 +28,10 @@ import (
 	helmv3 "github.com/fluxcd/helm-operator/pkg/helm/v3"
 	v3 "github.com/fluxcd/helm-operator/pkg/helm/v3"
 	daemonhttp "github.com/fluxcd/helm-operator/pkg/http/daemon"
-	"github.com/fluxcd/helm-operator/pkg/operator"
 	"github.com/fluxcd/helm-operator/pkg/release"
 	"github.com/fluxcd/helm-operator/pkg/status"
 	"github.com/fluxcd/helm-operator/pkg/utils"
+	"github.com/fossas/helm-operator/pkg/operator"
```

Comment:
Here and elsewhere, why are we now using the forked version of `pkg/operator`? I don't see any material changes to that package.

Reply:
They import an internal locking package and I can't for the life of me get Go to build without changing this import. I've tried multiple incantations of replace directives and have had no luck.